setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name = "OpenFisca-France",
-    version = "74.1.1-beta.1",
+    version = "74.1.1-beta.4",
```

Comment:
Do we really need to publish a new tagged version if we only change CI stuff? 🤔

Reply:
I am not sure. We did when we restructured the CircleCI config file.